### PR TITLE
Merge master -> 1.0.x

### PR DESCRIPTION
**Pull request by mwaggett**

### CODE REVIEW

File: lib/packaging/platforms.rb [hunk 1]
```diff
@@ -41,8 +41,10 @@ module Pkg::Platforms # rubocop:disable Metrics/ModuleLength
     'fedora' => {
       'f24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       'f25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
+      'f26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       '24' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
       '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' }
```
Comment (underscorgan) [inline lib/packaging/platforms.rb]: Missing trailing comma here

File: lib/packaging/platforms.rb [hunk 2]
```diff
@@ -41,8 +41,10 @@ module Pkg::Platforms # rubocop:disable Metrics/ModuleLength
     'fedora' => {
       'f24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       'f25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
+      'f26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       '24' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
-      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' }
+      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
+      '26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', }
```
Comment (demophoon) [inline lib/packaging/platforms.rb]: Would it be worthwhile to put a comma at the end of this so that the next time we add a new platform we don't dirty lines of unrelated changes?

**Comment by melissa** [comment]
we might want a commit with a slightly more descriptive commit message than `fix`. Also, how did that get dropped? Was that during merge conflict fixing?

File: lib/packaging/platforms.rb [hunk 3]
```diff
@@ -41,8 +41,10 @@ module Pkg::Platforms # rubocop:disable Metrics/ModuleLength
     'fedora' => {
       'f24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       'f25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
+      'f26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       '24' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
-      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' }
+      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
+      '26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4' }
```
Comment (melissa) [inline lib/packaging/platforms.rb]:
```
      'fedora' => {
        'f24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
        'f25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
        'f26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
        '24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
        '25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
        '26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
      },
```
the master branch has commas after all these entries

Comment (melissa) [reply lib/packaging/platforms.rb]: wow, it looks like these two files are very different in the two branches. That's not good

**Comment by mwaggett** [comment]
@melissa Yeah, I forgot that rebasing isn't easy with mergeups - fixed the commit messages (let me know if they still suck).

**Comment by mwaggett** [comment]
@demophoon @underscorgan @melissa added comma to last entry

File: lib/packaging/platforms.rb (hunk 3 above)
Comment (mwaggett) [reply lib/packaging/platforms.rb]: I think you did a lot of reorganization when you started this branch.

**Comment by demophoon** [comment]
Want to squash these?

**Comment by mwaggett** [comment]
@demophoon squashed

File: lib/packaging/platforms.rb [hunk 4]
```diff
@@ -41,8 +41,10 @@ module Pkg::Platforms # rubocop:disable Metrics/ModuleLength
     'fedora' => {
       'f24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       'f25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
+      'f26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       '24' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
-      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' }
+      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
+      '26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4' },
```
Comment (melissa) [inline lib/packaging/platforms.rb]:
```
        '25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
        '26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
```
there should also be commas after `'v4'`

File: lib/packaging/platforms.rb [hunk 5]
```diff
@@ -41,8 +41,10 @@ module Pkg::Platforms # rubocop:disable Metrics/ModuleLength
     'fedora' => {
       'f24' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
       'f25' => { :architectures => ['i386', 'x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
-      '24' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' },
-      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4' }
+      'f26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
+      '24' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4', },
+      '25' => { architectures: ['i386', 'x86_64'], repo: true, package_format: 'rpm', signature_format: 'v4', },
+      '26' => { :architectures => ['x86_64'], :repo => true, :package_format => 'rpm', :signature_format => 'v4', },
```
Comment (mwaggett) [inline lib/packaging/platforms.rb]: @melissa commas added

Comment (melissa) [reply lib/packaging/platforms.rb]: awesome, this should make future merge ups less painful, since there are fewer differences between master and 1.0.x
